packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -11,8 +11,8 @@ import locationValidator from '../../helpers/locationValidator'
 const PlaceholderSVG = styled.svg`
   background: no-repeat center / cover url('https://static.zooniverse.org/www.zooniverse.org/assets/fe-project-subject-placeholder-800x600.png');
   touch-action: pinch-zoom;
-  max-width: ${props => props.maxWidth || '100%'};
-  ${props => props.maxHeight && css`max-height: ${props.maxHeight};`}
+  max-width: ${props => props.$maxWidth || '100%'};
+  ${props => props.$maxHeight && css`max-height: ${props.$maxHeight};`}
```

Comment:
The addition of `$` is a [transient props](https://styled-components.com/docs/faqs#transient-props-since-5.1) strategy to avoid the warning generated by passing non-valid HTML attributes to the element. We rarely do this in FEM, so this change is unique to SingleImageViewer in this PR.